ee/clickhouse/queries/retention/retention_event_query.py

```diff
@@ -21,10 +22,29 @@ class RetentionEventsQuery(ClickhouseEventQuery):
     _event_query_type: RetentionQueryType
     _trunc_func: str
 
-    def __init__(self, event_query_type: RetentionQueryType, *args, **kwargs):
+    def __init__(
```

Comment:
Q: Why is this change needed? This is unrelated to stickiness and creates a lot of noise in the code.